components/symbl_ai/actions/get-job-status/get-job-status.mjs

```diff
@@ -0,0 +1,32 @@
+import symblAIApp from "../../symbl_ai.app.mjs";
+
+export default {
+  key: "symbl_ai-get-job-status",
+  name: "Get Job Status",
+  description: "Get the status of an Async job request. See the doc [here](https://docs.symbl.ai/docs/async-api/overview/jobs-api#http-request)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    symblAIApp,
+    jobId: {
+      type: "string",
+      label: "Job Id",
+      description: "The Id of the job request",
+      optional: false,
+    },
```

Comment:
**Note**
Typically, app-specific props that may be reused in future components should be added as a prop definition in the app file (see [guideline docs](https://pipedream.com/docs/components/guidelines/#prop-definitions)).

The [GET Job Status](https://docs.symbl.ai/docs/async-api/overview/jobs-api#get-job-status) endpoints appears to be the only one that takes `jobId` as a parameter, so it seems like it won't be reused in future components.

Reply:
@js07 You are right. This is the only action or API that makes use of the `jobId` as a parameter. Thanks!